pkg/identity/hsmsign_test.go

```diff
@@ -0,0 +1,134 @@
+// +build pkcs11
+
+/*
+Copyright 2020 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package identity
+
+import (
+	"errors"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func findSoftHSMLibrary() string {
```

Comment:
This could take a `t *testing.T` parameter and use this to abort the test cleanly if the library is not found